ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -121,6 +120,25 @@ class AlertsView extends React.Component {
     return pages;
   };
 
+  composeParams = (id, page, framework, status) => {
+    const params = id
+      ? { id }
+      : { framework: framework.id, page, status: summaryStatusMap[status] };
+
+    // created for UI purposes but not valid as API parameter
+    const doNotFilter = -1;
```

Comment:
This comment refers to the `doNotFilter` value?

Reply:
No. Will change its position.

Reply:
Changed.